kafka/signed_token_redeem_handler.go

```diff
@@ -91,10 +93,12 @@ func SignedTokenRedeemHandler(
 		return nil
 	}
 
-	// Create a lookup for issuers & signing keys based on public key
+	// Create a lookup for issuers & signing keys based on public key.
 	signedTokens := make(map[string]SignedIssuerToken)
+	now := time.Now()
+
 	for _, issuer := range issuers {
-		if !issuer.ExpiresAtTime().IsZero() && issuer.ExpiresAtTime().Before(time.Now()) {
```

Comment:
There is absolutely no need to:
- spread this piece of logic across multiple files;
- calculate expiry time two times each issuer;
- calculate `now` on each iteration.